controllers/admin/AdminTranslationsController.php

```diff
@@ -1275,7 +1275,7 @@ protected function userParseFile($content, $type_translation, $type_file = false
             case 'modules':
                 // Parsing modules file
                 if ($type_file == 'php') {
-                    $regex = '/->l\((\')' . _PS_TRANS_PATTERN_ . '\'(, ?\'(.+)\')?(, ?(.+))?\)/U';
+                    $regex = '/->l\(\s*(\')' . _PS_TRANS_PATTERN_ . '\'(\s*,\s*?\'(.+)\')?(\s*,\s*?(.+))?\s*\)/U';
```

Comment:
You can also add `ms`  (http://php.net/manual/en/reference.pcre.pattern.modifiers.php)

Reply:
The modifiers are now set to Ums in a6e8796ba1356ce6ade0b9a745b81be22032bcc0